test/libsolidity/syntaxTests/storageLayoutSpecifier/immutables.sol

```diff
@@ -0,0 +1,10 @@
+contract A {
+    address immutable a = 0x0000000000000000000000000000000000000001;
+    uint immutable x = 1;
```

Comment:
Please also add an immutable that's not pure (i.e. one that's not initialized with a literal) since that's a separate case. Also a comment explaining how they differ.